### PR TITLE
zack's layering enhancement updates

### PR DESCRIPTION
**Pull request by cheesesashimi**

### CODE REVIEW

File: enhancements/ocp-coreos-layering/ocp-coreos-layering.md [hunk 1]
```diff
@@ -115,6 +133,10 @@ Acme Corp has 300 clusters distributed across their manufacturing centers. They
 
 In the future, we may additionally support a signature mechanism (IMA/fs-verity) that allows these images can be signed centrally.  Each node verifies this image rather than generating a custom state.
 
+The build and test workflow could resemble something like this:
```
Comment (sinnykumari) [inline enhancements/ocp-coreos-layering/ocp-coreos-layering.md]: Should we move this to `### External image build workflow section`?

**Comment by sinnykumari** [comment]
Thanks Zack for the PR.

One small inline change. Also, can you squash all the commits into a single commit and update commit message accordingly?

File: enhancements/ocp-coreos-layering/ocp-coreos-layering.md (hunk 1 above)
Comment (cheesesashimi) [reply enhancements/ocp-coreos-layering/ocp-coreos-layering.md]: Yeah, it would be better there. I've made the change.

**Comment by cheesesashimi** [comment]
Changes made and commits squashed.

**Pull request merged**